exec/vector/src/main/java/org/apache/drill/exec/vector/DateUtilities.java

```diff
@@ -195,7 +197,27 @@ public static int timeToMillis(int hours, int minutes, int seconds, int millis)
    * @param localTime Java local time
    * @return Drill form of the time
```

Comment:
@paul-rogers 
Would you mind please adding a brief javadoc to this class explaining what a Drill form of the time is?